admin/app/models/certification.js

```diff
@@ -23,6 +23,7 @@ export default class Certification extends Model {
   @attr() competencesWithMark;
   @attr('boolean', { defaultValue: false }) isPublished;
   @attr('boolean', { defaultValue: false }) isV2Certification;
+  @attr('string') hasAcquiredCleaCertification;
```

Comment:
Pas besoin de préciser quelle transform appliquer à la réception, Ember s'en sort tout seul.

```suggestion
  @attr() hasAcquiredCleaCertification;
```
ça suffit largement.